### PR TITLE
Add an RPC deleteAllPendingTransactions()

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: integration_tests/Rpc.spec.ts [hunk 1]
```diff
@@ -397,6 +397,12 @@ describe("rpc", () => {
             });
         });
 
+        describe("with pending transactions", () => {
```
Comment (joojis) [inline integration_tests/Rpc.spec.ts]: There already is `describe("with pending transactions", ...)` right below.

**Comment by junha1** [comment]
@joojis `"message": "Method not found"` How to update Codechain and reflect here?

**Comment by joojis** [comment]
@junha1 You need to update the codechain docker tag in https://github.com/CodeChain-io/codechain-sdk-js/blob/master/.travis.yml

As far as I remember, You can create a docker tag by pushing the revision to the `docker-build` branch of the codechain repo.

If it's created, the tag must be shown [here](https://hub.docker.com/r/kodebox/codechain/tags/)

**Comment by joojis** [comment]
@majecty Do we still maintain the docker-build branch? I'm not sure because I haven't used it for a while.

**Comment by majecty** [comment]
@joojis I think the "docker-build" branch is useful to create a docker image. We can use it.

**Comment by junha1** [comment]
@joojis @majecty Updated

**Pull request merged**